src/escape.d

```diff
@@ -272,18 +276,20 @@ bool checkEscapeRef(Scope* sc, Expression e, bool gag)
         {
             assert(d);
             VarDeclaration v = d.isVarDeclaration();
-            if (v && v.toParent2() == sc.func)
+            if (v)
```

Comment:
Picture you didn't write this code but need to review it. So we have an `if` statement here controlling code that's literally two pages tall. Presumably something is going on after the `if`, so the reviewer needs to either jump to the end to figure whether they need to remember it (as I do), or make a mental note that the entire code is qualified by a huge-ass `if`.

Compare with the much simpler:

```d
if (!v)
{
    return;
}
... ah, so _all_ that follows is only applicable to v being non-null ...
...
```